### DISCUSSION TITLE
which is the best voice model you have experienced now

### DISCUSSION THREAD

**Discussion by LinJinghuidev** [coqui-ai/TTS#843] [General Q&A]
I use fastspeech 2 to train the baker. The audio will have mechanical sound. I want to know which is the best voice model you have experienced now

**Comment by Lizh-TW** [comment]
![image](https://user-images.githubusercontent.com/58511222/135016697-f99ccee6-a08e-4f8c-81c4-923b1370acb5.png)
Now vits are fixed, but Glow-TTS maybe have some problems, because when I trained flow recipes get an error https://github.com/coqui-ai/TTS/issues/822 .

**Comment by Sadam1195** [comment]
> I use fastspeech 2 to train the baker. The audio will have mechanical sound. I want to know which is the best voice model you have experienced now

Purely, in terms of quality, I haven't seen better results than Tacotron2 DDC + WaveRNN.
@LinJinghuidev
